src/server/system_services/bucket_server.js

```diff
@@ -754,10 +754,6 @@ function check_for_lambda_permission_issue(req, bucket, removed_accounts) {
 async function delete_bucket_and_objects(req) {
     var bucket = find_bucket(req);
 
-    // fail on namespace bucket
```

Comment:
`delete_bucket_and_objects ` is called from the OBC delete flow. I feel that in the case of namespace it is not well defined.
what will happen in different namespace types?

Reply:
correct me if I missed something - bucket reclaimer does:
1. find deleting buckets
2. search in MD_store for objects of the deleting bucket and deleted them.
3. search for the bucket's tiers and tiering policy 
4. remove the bucket from the allowed buckets permission lists of accounts  

if i'm not mistaken the only different step is 2 and I can skip it if the bucket is namespace bucket...
WDYT?

Reply:
@romayalon that sounds right. just make sure to test this flow